### PR TITLE
Remove locust dependecies from helper code.

### PR DESCRIPTION
**Pull request by partickl**
Add new testcases that runs without locust

### CODE REVIEW

File: testCases/casual/casual/performance/test/configuration.py [hunk 1]
```diff
@@ -535,6 +535,11 @@ def __init__(self):
 # configuration configuration #
 ###############################
 
+def non_null_vars(x):
```
Comment (ahhud) [inline testCases/casual/casual/performance/test/configuration.py:538]: Nice!

File: testCases/casual/casual/performance/test/casual.py [hunk 2]
```diff
@@ -92,13 +82,32 @@ def information( configuration, environment):
                 "commit": reply["result"]["version"]["commit"]
             }
 
+def scale_queue_forward(configuration, domain_name, alias, instances):
```
Comment (ahhud) [inline testCases/casual/casual/performance/test/casual.py:85]: Nice!

File: testCases/casual/001.single.service.scale.payload.py [hunk 3]
```diff
@@ -36,31 +33,31 @@ class TestCase( FastHttpUser):
     def task1( self):
         self.client.post(
             name = "1B",
-            url = "/casual/example/echo",
+            path = "/casual/example/echo",
```
Comment (ahhud) [inline testCases/casual/001.single.service.scale.payload.py]: We need to decide which version of locust we shall support in this path/url turmoil.

Comment (partickl) [reply testCases/casual/001.single.service.scale.payload.py]: why not choose neither? if whe keep it as the first parameter we can omit the name, and once we've settled on a version we can reintroduce the named parameter if we want to

Comment (partickl) [reply testCases/casual/001.single.service.scale.payload.py]: removed path/url. tested ok with locust 2.8.6 and 2.24.1

**Review by ahhud — APPROVED** [review]

**Pull request merged**